python/instrumentation/openinference-instrumentation-langchain/tests/openinference/instrumentation/langchain/test_instrumentor.py

```diff
@@ -43,7 +43,9 @@
 @pytest.mark.parametrize("is_async", [False, True])
 @pytest.mark.parametrize("is_stream", [False, True])
 @pytest.mark.parametrize("status_code", [200, 400])
+@pytest.mark.parametrize("use_context_attributes", [False, True])
```

Comment:
I understand this test case is already doing this, but if adding a parameter to a test case requires adding complex branching logic to the test case, it should not be another parameter in the test case, but should be a separate test case.

Reply:
Done